gensrc/proto/internal_service.proto

```diff
@@ -257,6 +257,64 @@ message PTabletWriterCancelResult {
 message PExecPlanFragmentRequest {
 };
 
+message PTabletReaderOpenRequest {
+    optional PUniqueId id = 1;
+    optional int64 tablet_id = 2;
+    optional int64 version = 3;
+}
+
+message PTabletReaderOpenResult {
+    optional StatusPB status = 1;
```

Comment:
Why is it not `required` as that in `PTabletReaderMultiGetResult`?

Reply:
ok,  all will change to optional

Reply:
according to new rule, all new pb should use optional, required is forbidden, and proto3 has removed required